xml/en/docs/http/ngx_http_upstream_module.xml

```diff
@@ -905,7 +905,7 @@ The <literal>least_time</literal> method is available as a part of our
     [<literal>status_zone</literal>=<value>zone</value>]</syntax>
```

Comment:
This now needs a distinct note for the "status_zone" parameter commercial status, similar to the "resolver" directive in the core module.